submissions/kriLEBED/html_css_popup/index.html

```diff
@@ -0,0 +1,149 @@
+                                <label for="more"></label>
+                                <input type="checkbox" id="more" class="more_btn"/>
+                                <ul class="submenu-items more-submenu">
+                                    <li class="submenu-item">
+                                        <a class="submenu-link" href="#">
+                                            <img src="img/settings-icon.png" alt="settings">
+                                            <p>Settings</p>
+                                        </a>
+                                    </li>
+                                    <li class="submenu-item">
+                                        <a class="submenu-link" href="#">
+                                            <img src="img/weather-icon.png" alt="weather">
+                                            <p>Weather</p>
+                                        </a>
+                                    </li>
+                                    <li class="submenu-item">
+                                        <a class="submenu-link" href="#">
+                                            <img src="img/calculator-icon.png" alt="calculator">
+                                            <p>Calculator</p>
+                                        </a>
+                                    </li>
+                                    <li class="submenu-item">
+                                        <a class="submenu-link" href="#">
+                                            <img src="img/download-icon.png" alt="download">
+                                            <p>Download</p>
+                                        </a>
+                                    </li>
+                                    <li class="submenu-item">
+                                        <a class="submenu-link" href="#">
+                                            <img src="img/chat-icon.png" alt="chat">
+                                            <p>Chat</p>
+                                        </a>
+                                    </li>
+                                    <li class="submenu-item">
+                                        <a class="submenu-link" href="#">
+                                            <img src="img/wallet-icon.png" alt="wallet">
+                                            <p>Wallet</p>
+                                        </a>
+                                    </li>
+                                </ul>
+                            </div>
+                        </div>
+                    </li>
+                    <li class="menu_items"><a href="#"><img src="img/bell-icon.png" alt="bell"></a></li>
+                    <li class="menu_items"><a href="https://github.com/kriLEBED"><img src="img/photo.jpg" alt="profile" class="profile-img"></a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+</body>
+</html>
```

Comment:
if you see this symbol at the end of the file on Github - it means that the empty line is needed after the last tag (or any  last symbol at your file)
![empty line](https://user-images.githubusercontent.com/10871447/101679509-22d00500-3a68-11eb-8031-1416f4ff9156.png)
https://github.com/kottans/frontend-2021-homeworks/pull/254/files
Could you please add the empty line (without spaces)